### ISSUE TITLE
`Report.__init__() got an unexpected keyword argument 'descriptor'` when creating a frictionless validation report

### ISSUE THREAD

**Issue by fjuniorr** [frictionlessdata/livemark#230]
In versions `frictionless==4.29.0` and `livemark==0.88.0` I could create a [frictionless validation report](https://livemark.frictionlessdata.io/pages/feature-reference/markdown.html#report) with

````
```yaml report
descriptor: validation-report.json
```
````

There is a working example [here](https://github.com/transparencia-mg/validacao-leiaute). 

Now, with `frictionless==5.4.0` and `livemark==0.110.2` when trying to build that same type of validation report (but using the v5 version of report) I get:

```
Report.__init__() got an unexpected keyword argument 'descriptor'
```

I did not found any mentions of breaking changes in the [release page](https://livemark.frictionlessdata.io/pages/release.html).

**Comment by roll** [comment]
Thanks! We will fix it.

**Comment by roll** [comment]
Thnks, it must be fixed in `livemark@0.110.4`
